packages/conferer/package.yaml

```diff
@@ -5,6 +5,7 @@ synopsis: |
 description: |
   Library to abstract the parsing of many haskell config values from different config sources
 
+
```

Comment:
this whitespace changes nothing, can you remove it?

Reply:
fixed